### ISSUE TITLE
Add link handling for images/videos and browser option

### ISSUE THREAD

**Issue by jordan1776** [liftoff-app/liftoff#39]
### Is your feature request related to a problem? Please describe.

Most links that might be better to open in other app, open in Liftoff

### Describe the solution you'd like

There should be an option in settings to check off what links should open in app (instance images, youtube, streamable, gfycat, etc). And another option for opening websites in the internal browser, custom tabs, or the actual browser (with a setting for domains that should always open in app (mastodon, spotify, etc))

### Additional context

Internal browser should have swipe back like the comments view.

**Comment by kevinross** [comment]
(not OP, but want to add a modification to the request that's not incompatible with the request as a whole) 

for links to external sites, while it'd be nice to have a global setting for how to open them every time, i think the first thing to do would be to implement it for the case-by-case scenario: if I open a link but want to come back to it later after reading the first paragraph or so, I can't do so directly (I'd have to find the URL, then punch it into my address bar). As it is now, I can't find a way to open the site in an external browser (come to think of it, I can't find _any_ UI elements in the embedded browser: no "open in external", no "switch to desktop view", no "send to...", nothing).

**Comment by Generator** [comment]
Duplicate of #28

**Comment by jordan1776** [comment]
Not exactly, #28 is about opening all links in external apps (browser and other apps), mine is more about opening links in internal browser and twitter links in app. When the internal browser becomes more usable this way would probably be come more preferred.
